tests/validation/tests/v3_api/test_cluster_templates.py

```diff
@@ -609,15 +758,15 @@ def cluster_template_create_edit(userToken):
     cluster_cleanup(client, cluster)
 
 
-def node_template_do(userclient):
+def node_template_do(userclient, nodesize):
     client = userclient
     do_cloud_credential_config = {"accessToken": DO_ACCESSKEY}
     do_cloud_credential = client.create_cloud_credential(
         digitaloceancredentialConfig=do_cloud_credential_config)
     time.sleep(3)
     node_template = client.create_node_template(
         digitaloceanConfig={"region": "nyc3",
-                            "size": "4gb",
+                            "size": nodesize,
                             "image": "ubuntu-16-04-x64"},
```

Comment:
Can we use Ubuntu 18.03?

Reply:
Updated to Ubuntu 18.04. Also, updated ubuntu version and engineinstall url in rke_cluster_provisioning.py